app/src/main/java/me/quadphase/qpdex/databaseAccess/PokemonFactory.java

```diff
@@ -390,7 +406,7 @@ private void minimalPokemonNameInitializer(Cursor cursor){
 
 
     public void getAllDetailedPokemon(){
-        //NOTE: This method is here for testing purposes. Avoid using in production builds.
```

Comment:
This comment should be preserved since it is unlikely we'll build all Pokemon objects in one shot.